components/navigationBar/index.html

```diff
@@ -1,12 +1,12 @@
 <div>
-    <button v-if="back === true" class="back nav-btn" v-on="click: goBack">&lt;</button>
-    <a v-if="typeof back === 'string'" class="back nav-btn" href="{{back}}">&lt;</a>
+    <button v-if="back === true" class="back nav-btn" v-on="click: goBack"><span class="icon ion-android-arrow-back"></span></button>
+    <a v-if="typeof back === 'string'" class="back nav-btn" href="{{back}}"><span class="icon ion-android-arrow-back"></span></a>
 
     <button v-if="cancel === true" v-on="click: goBack" class="nav-btn">{{'Cancel' | i18n}}</button>
     <a v-if="typeof cancel === 'string'" href="{{cancel}}" class="nav-btn">{{'Cancel' | i18n}}</a>
 </div>
 
-<h1>{{title | i18n}}</h1>
+<h1>{{app.name}}</h1>
```

Comment:
If you really think this is gross, which it is, you can alternatively change this line in share/index.html https://github.com/mozilla/webmaker-app/blob/master/views/share/index.html#L1
```
<nav v-component="navigationBar" class="nav-default" v-with="title: app.name"></nav>
```

That will explicitly pass the `app.name` value along as the `title` property in navigationBar.